tests/test.yml

```diff
@@ -13,6 +13,25 @@
         - sudo
         - which
       when: ansible_os_family == 'RedHat'
+    # adding this to the host allows us to properly resolve the KDC and test it on vagrant - does not work on Docker
```

Comment:
I don't think that this is not necessary for Vagrant. If that's the case, please remove the comment.